cmds/core/pci/pci_linux_test.go

```diff
@@ -43,8 +43,13 @@ func TestRun(t *testing.T) {
 		},
 	} {
 		t.Run(tt.name, func(t *testing.T) {
-			c := command(io.Discard, tt.args...)
-			c.run()
+			c, err := command(io.Discard, tt.args...)
+			if err != nil {
+				t.Errorf("got %v, want nil", err)
+			}
+			if err := c.run(); err != nil {
+				t.Errorf("got %v, want nil", err)
+			}
```

Comment:
The test for PCI fail for the cases 2,3,4:

```
=== RUN   TestRun
=== RUN   TestRun/switch_hexdump_case_1
=== RUN   TestRun/switch_hexdump_case_2
    pci_linux_test.go:51: got permission denied, want nil
=== RUN   TestRun/switch_hexdump_case_3
    pci_linux_test.go:51: got permission denied, want nil
=== RUN   TestRun/switch_hexdump_case_4
    pci_linux_test.go:51: got permission denied, want nil
=== RUN   TestRun/readJSON_true,_without_error
=== RUN   TestRun/readJSON_true,_error_in_os.ReadFile
=== RUN   TestRun/readJSON_true,_error_in_json.Unmarshal
=== RUN   TestRun/dumpJSON
=== RUN   TestRun/invoke_registers
--- FAIL: TestRun (0.03s)
```

Again, permission denied errors indicate that we should move this to vmtest root permissioned integration tests.